modules/react/combobox/lib/ComboboxMenuItem.tsx

```diff
@@ -57,17 +61,37 @@ export const useComboboxMenuItem = composeHooks(
   useListItemRegister,
   useListItemAllowChildStrings
 );
+const comboboxMenuItemIconStencil = createStencil({
+  base: {
+    alignSelf: 'start',
+  },
+});
+
+export const ComboboxMenuItemIcon = createSubcomponent('span')({
+  modelHook: useComboboxModel,
+  subComponents: {
+    Icon: styled(SystemIcon)({alignSelf: 'start'}),
```

Comment:
Should we be using `styled` here? Or are we trying to step away from this?

Reply:
Removed this